uber/config.py

```diff
@@ -639,15 +639,8 @@ def HTTP_METHOD(self):
     def get_kickin_count(self, kickin_level):
         from uber.models import Session, Attendee
         with Session() as session:
-            attendees = session.query(Attendee)
-            individual_supporters = attendees.filter(
-                Attendee.paid.in_([self.HAS_PAID, self.REFUNDED]),
-                Attendee.amount_extra >= kickin_level).count()
-            group_supporters = attendees.filter(
-                Attendee.paid == self.PAID_BY_GROUP,
-                Attendee.amount_extra >= kickin_level,
-                Attendee.amount_paid >= kickin_level * 100).count()
-            return individual_supporters + group_supporters
+            count = session.query(Attendee).filter_by(amount_extra=kickin_level).count()
```

Comment:
We'd generally rather count high than low, so I simplified this function significantly.